lib/rubocop/rspec/language/node_pattern.rb

```diff
@@ -7,6 +7,8 @@ module Language
       module NodePattern
         extend RuboCop::NodePattern::Macros
 
+        def_node_matcher :rspec?, '{(const {nil? cbase} :RSpec) nil?}'
```

Comment:
For some reason, I had an idea that this is impossible.

But is it good?
Using `#{RSPEC}` in several node patterns mean we'll compiling this part into Ruby code each time. For `#rspec`, we're compiling once and reusing the already compiled code across other node matchers that use it.

Reply:
> For some reason, I had an idea that this is impossible.

Neither did I :joy: